src/Controls/src/Core/MessagingCenter.cs

```diff
@@ -202,9 +202,9 @@ void InnerSend(string message, Type senderType, Type argType, object sender, obj
 			if (message == null)
 				throw new ArgumentNullException(nameof(message));
 			var key = new Sender(message, senderType, argType);
-			if (!_subscriptions.ContainsKey(key))
+			if (!_subscriptions.TryGetValue(key, out List<Subscription> subcriptions))
```

Comment:
```suggestion
			if (!_subscriptions.TryGetValue(key, out List<Subscription> subscriptions))
```

Reply:
and fix below

Reply:
I was debating whether to fix the typos or keep the diff minimal. Your suggestion is probably the right one.